tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.html

```diff
@@ -557,7 +562,10 @@
                     // after the transition ends.
                         .attr("d", pathDataFn);
                     window.setTimeout(
-                        ()=>foregroundPaths.each(setPathControlPointsFn),
+                        ()=>{
+                          foregroundPaths.each(setPathControlPointsFn),
```

Comment:
This should be a semicolon, not a comma.

Reply:
Thanks. Done!